sympy/core/relational.py

```diff
@@ -237,6 +237,25 @@ def canonical(self):
 
         return r
 
+    def applyfunc(self, op):
+        """Apply the same function to both sides of the equation.
+        Calls the function for each side, passing the lhs/rhs as
+        an argument, and then returns a new equation consisting of
+        the returned values."""
```

Comment:
This should follow the doc guidelines for docstrings:
https://docs.sympy.org/latest/documentation-style-guide.html

I think probably all that is needed here is a short one line description like "Apply op to both sides of the relation" and then a doctest example.

This is for all Relational though, not just Equality so it shouldn't say equation.